rvv-intrinsic-generator/golden/api-testing/vaadd.c

```diff
@@ -0,0 +1,809 @@
+#include <stdint.h>
+#include <riscv_vector.h>
+
+#ifndef FLMUL
+#define FLMUL 0
+#endif
```

Comment:
The motivation of api-testing folder is used to testing gnu toolchain which are not support fractional LMUL before. We need to remove define FLMUL if gcc supports fractional LMUL now.